plugins/experimental/cert_reporting_tool/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+#######################
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+#  agreements.  See the NOTICE file distributed with this work for additional information regarding
+#  copyright ownership.  The ASF licenses this file to you under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with the License.  You may obtain
+#  a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing permissions and limitations under
+#  the License.
+#
+#######################
+
+project(cert_reporting_tool)
+
+add_atsplugin(cert_reporting_tool cert_reporting_tool.cc)
+
+target_include_directories(cert_reporting_tool PRIVATE "${PROJECT_SOURCE_DIR}/include")
+
+target_link_libraries(cert_reporting_tool
+    PRIVATE
+        libswoc
```

Comment:
Why does it need libswoc?

Reply:
Oh, I saw TextView in the include and didn't notice it was coming from tscpp instead of libswoc. I'll remove libswoc.